u-boot/Makefile

```diff
@@ -274,7 +274,7 @@ include ../build.conf
 
 ifeq ($(PLATFORM),$(filter $(PLATFORM),virt64))
 ARCH	= arm64
-CROSS_COMPILE = aarch64-linux-gnu-
+CROSS_COMPILE = aarch64-none-linux-gnu-
```

Comment:
Base on the SO3 doc, the original compiler (without `none`) is correct
--> don't push/merge this change.

Reply:
To be discussed further

Reply:
Will be corrected in future issue